### PR TITLE
Framework for backend added

### PR DESCRIPTION
**Pull request by sterlingwalker**
- Backend initialized so that team is able to work on certain sections in next sprint

- A lot of the networking URLs are just placeholders since I have to initialize the database

### CODE REVIEW

File: src/main/java/com/management/HumanResources/controller/ReadController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    @Autowired private FirebaseDao firebase;
+
+    @GetMapping(path = "/employees")
+    public List<Employee> getEmployees() {
+        return firebase.getAllEmployees();
+    }
+
+    @GetMapping(path = "/employee/{id}")
+    public Employee getSpecificEmployee(@PathVariable int id) {
```
Comment (yonimn2000) [inline src/main/java/com/management/HumanResources/controller/ReadController.java]: I feel like `getEmployee` would be sufficient, but whatever you prefer.

File: src/main/java/com/management/HumanResources/dao/FirebaseDao.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            .build();
+  }
+
+  //Database paths are just placeholders until the actual ones are created
+
+  public List<Employee> getAllEmployees() {
+      return getListOfObjects(Employee.class, "path");
+  }
+
+  public Employee getSingleEmployee(int id) {
```
Comment (yonimn2000) [inline src/main/java/com/management/HumanResources/dao/FirebaseDao.java]: I feel like `getEmployee` would be sufficient, but whatever you prefer.

File: src/main/java/com/management/HumanResources/dao/FirebaseDao.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      return getSingleObject(Employee.class, "path/{employeeId}", id);
+  }
+
+  public String updateSingleEmployee(Employee employee) {
```
Comment (yonimn2000) [inline src/main/java/com/management/HumanResources/dao/FirebaseDao.java]: same here and the rest of the relevant methods in this file

**Review by yonimn2000 — CHANGES_REQUESTED** [review]
That is a good framework for everyone to start working on the backend. 
I have left a couple of comments in this review.

File: src/main/java/com/management/HumanResources/dao/FirebaseDao.java (hunk 3 above)
Comment (sterlingwalker) [reply src/main/java/com/management/HumanResources/dao/FirebaseDao.java]: Good Suggestions 👍

**Comment by yonimn2000** [comment]
Perfect!
Thanks!

**Pull request merged**